examples/auth_example/auth_example_client/lib/src/protocol/client.dart

```diff
@@ -30,10 +34,16 @@ class EndpointExample extends _i1.EndpointRef {
 
 class Modules {
   Modules(Client client) {
-    auth = _i3.Caller(client);
+    serverpod_auth_email = _i3.Caller(client);
+    serverpod_auth_email_endpoint = _i4.Caller(client);
+    auth = _i5.Caller(client);
   }
 
-  late final _i3.Caller auth;
+  late final _i3.Caller serverpod_auth_email;
```

Comment:
TBD; would be nice not to have this, if no endpoints are defined, but since this is a straight important, it would require code-gen in this project if the dependency adds or removes an endpoint (at least for changes between 0 and !0).